modules/stack/variables.tf

```diff
@@ -340,3 +340,9 @@ variable "space_name" {
   description = "If using a dedicated space, override the name of the space (instead of using `component_name`)."
   default     = null
 }
+
+variable "use_depends_on_resource" {
```

Comment:
```suggestion
variable "spacelift_stack_dependency" {
```

Reply:
In the interest of time, I'll use the prior suggestion for both of these... I think that's what works best for now: `spacelift_stack_dependency_enabled`